src/interfaces/IQuoter.sol

```diff
@@ -0,0 +1,63 @@
+pragma solidity ^0.8.0;
+
+struct ExtraData {
+    uint64 token0PrecisionMultiplier;
+    uint64 token1PrecisionMultiplier;
+    uint64 amplificationCoefficient;
+}
+
+interface IQuoter {
+
+    /*//////////////////////////////////////////////////////////////////////////
+                                QUERY METHODS (VIEW)
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @param aExtraData for StablePair use, to leave blank for ConstantProductPair. See ReservoirLibrary
```

Comment:
Natspec is not accurate, did you mean to use inherit for base params, or does this need to be filled out?

Reply:
The part about the ReservoirLibrary is no longer relevant so I removed that (since it's moved into the `IQuoter` interface). 

`IQuoter` is already the base class so there's nothing more "basic" to inherit the docs from. I was thinking, all the other parameters are pretty self-explanatory so no need for explanation. Only the `aExtraData` is worth explaining. 

What do you think? Or do you think we need to document all function params?

Reply:
okay fixed the natspec field